_overviews/scala3-book/scala-features.md

```diff
@@ -63,7 +63,7 @@ val newNumbers = double(oldNumbers)
 ```
 
 That code instructs the compiler what to do on a step-by-step basis.
-Instead, we write high-level, functional code using higher-order functions and lambdas like this to achieve the same effect:
+Instead, we write high-level, functional code using higher-order functions and lambdas like this to compute the same result:
```

Comment:
I prefer “computing a result” over “achieving an effect”, which sounded more imperative to me.